### PR TITLE
change home route path name

### PR DESCRIPTION
**Pull request by LeoMurphy7**
resolve when conflict happen

### CODE REVIEW

File: src/components/home.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    this.state = { name: "", email: "", phone: "" };
+
+    this.handleChangeName = this.handleChangeName.bind(this);
+    this.handleChangeEmail = this.handleChangeEmail.bind(this);
+    this.handleChangePhone = this.handleChangePhone.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChangeName(event) {
```
Comment (MoeThet20) [inline src/components/home.jsx:18]: please make refactor again

File: src/components/home.jsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  handleSubmit(event) {
+    event.preventDefault();
+    alert(
+      "My name is " + this.state.name + this.state.email + this.state.phone
+    );
+  }
+
+  render() {
+    return (
+      <section className="card" style={{ margin: "0 auto" }}>
```
Comment (MoeThet20) [inline src/components/home.jsx:42]: remove style

File: src/components/navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
```
Comment (MoeThet20) [inline src/components/navbar.jsx:2]: ??

File: package.json [hunk 4]
```diff
@@ -3,13 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@mui/material": "^5.15.0",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.3.2",
```
Comment (MoeThet20) [inline package.json:12]: react-bootstrap

File: src/pages/login/login.jsx [hunk 5]
```diff
@@ -1,14 +1,35 @@
-import React from "react";
+import React, { useState } from "react";
+import { useAuth } from "../../utils/auth";
+import { useLocation, useNavigate } from "react-router";
+import "bootstrap/dist/css/bootstrap.min.css";
```
Comment (MoeThet20) [inline src/pages/login/login.jsx:4]: ??

File: src/pages/login/login.jsx [hunk 6]
```diff
@@ -1,14 +1,35 @@
...
+
   return (
-    <div style={{ display: "flex", flex: 1 }}>
-      <div style={{ width: "100vw", height: "100vh", position: "static" }}>
-        <img
-          src="/images/login.jpg"
-          style={{ objectFit: "cover", width: "100%", height: "100%" }}
-          alt="app bg icon"
-        />
+    <div className="card" style={{ width: "30rem" ,margin:" auto"}}>
```
Comment (MoeThet20) [inline src/pages/login/login.jsx:23]: remove inline style

File: src/pages/login/login.jsx [hunk 7]
```diff
@@ -1,14 +1,35 @@
...
+      <div className="card-body">
+        <h5 className="card-title">Login</h5>
+        <div style={{display:'flex',flexDirection:'row',marginBottom:"5px",alignItems:"center"}}>
```
Comment (MoeThet20) [inline src/pages/login/login.jsx:26]: use from bootstrap

File: src/pages/profile/profile.jsx [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  console.log('checking',auth.user)
+
+  const handleLogout = () => {
+    auth.logout();
+    navigate('/login')
+    
+  }
+
+  return (
+    <div style={{width:"400px", height:"200px" , margin:"0 auto"}}>
```
Comment (MoeThet20) [inline src/pages/profile/profile.jsx:17]: inner style

File: src/pages/sidebar/sidebar.jsx [hunk 9]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { useState } from "react";
+
+function Sidebar() {
+  const [inputs, setInputs] = useState({});
+  const handleChange = (event) => {
+    const name = event.target.name;
+    const value = event.target.value;
+    setInputs((values) => ({ ...values, [name]: value }));
+    console.log("checking",inputs )
```
Comment (MoeThet20) [inline src/pages/sidebar/sidebar.jsx:10]: remove console

File: src/App.js [hunk 10]
```diff
@@ -2,15 +2,24 @@ import "./App.css";
...
-     <div>
-      <h1>This is development branch</h1>
-     </div>
+    <AuthProvider>
+      <NavBar/>
+      <Routes>
+        <Route path="/login" element={<Login />}></Route>
+        <Route path="/sidebar" element={<Sidebar />}></Route>
+        <Route path="/homepage" element={<HomePage name={"Mustang"} />}></Route>
+          <Route path="/profile" element={<RequireAuth><Profile /></RequireAuth>}></Route>
```
Comment (MoeThet20) [inline src/App.js:19]: Public route,
Private route

File: src/App.js [hunk 11]
```diff
@@ -2,15 +2,24 @@ import "./App.css";
...
 
 function App() {
   return (
-     <div>
-      <h1>This is development branch</h1>
-     </div>
+    <AuthProvider>
+      <NavBar/>
+      <Routes>
+        <Route path="/login" element={<Login />}></Route>
```
Comment (MoeThet20) [inline src/App.js:16]: make route constant 
export const LOGIN = "/login";

File: package.json (hunk 4 above)
Comment (LeoMurphy7) [reply package.json:12]: done

File: src/components/home.jsx (hunk 1 above)
Comment (LeoMurphy7) [reply src/components/home.jsx:18]: done

**Pull request merged**